cms/plugin_rendering.py

```diff
@@ -123,6 +123,8 @@ def render_placeholder(placeholder, context_to_copy, name_fallback="Placeholder"
     if hasattr(placeholder, 'content_cache'):
         return mark_safe(placeholder.content_cache)
     page = placeholder.page if placeholder else None
+    site_id = get_site_id(getattr(page, 'site_id', None))
```

Comment:
please use a conditional, page can be none but if not then it will have a site_id attr.

``python
if page:
    site_id = page.site_id
else:
    site_id = get_site_id(None)
``